java/src/main/java/org/rocksdb/TableProperties.java

```diff
@@ -379,15 +391,6 @@ public String getCompressionName() {
     return readableProperties;
   }
 
-  /**
-   * The offset of the value of each property in the file.
-   *
-   * @return the offset of each property.
-   */
-  public Map<String, Long> getPropertiesOffsets() {
```

Comment:
Hi @adamretter - it seems that Java has getPropertiesOffsets() while c++ doesn't. Is it safe for me to delete that in this PR?